include/lkmm.h

```diff
@@ -1,45 +1,59 @@
 #include <stdint.h>
 
-typedef enum memory_order {
-  memory_order_relaxed,
-  memory_order_once,
-  memory_order_acquire,
-  memory_order_release,
-  mb,
-  wmb,
-  rmb,
-  rcu_lock,
-  rcu_unlock,
-  rcu_sync,
-  before_atomic,
-  after_atomic,
-  after_spinlock,
-  barrier,
-} memory_order;
-
-typedef enum operation {
-  op_add,
-  op_sub,
-  op_and,
-  op_or,
-} operation;
-
-extern int __LKMM_LOAD(void*, memory_order);
-extern void __LKMM_STORE(void*, int, memory_order);
-extern void __LKMM_FENCE(int);
-extern int __LKMM_XCHG(int*, int, memory_order);
-extern int __LKMM_CMPXCHG(int*, int, int, memory_order, memory_order);
-extern void __LKMM_ATOMIC_OP(int*, int, operation);
-extern int __LKMM_ATOMIC_FETCH_OP(int*, int, memory_order, operation);
-extern int __LKMM_ATOMIC_OP_RETURN(int*, int, memory_order, operation);
+typedef enum __LKMM_memory_order {
+  __LKMM_relaxed,
```

Comment:
LKMM does not have a `relaxed` mo. Based on [this](https://github.com/torvalds/linux/blob/4a95bc121ccdaee04c4d72f84dbfa6b880a514b6/tools/memory-model/linux-kernel.def#L36), all references to `relaxed` below needs to be changed to `once`.

Be aware of [this](https://github.com/hernanponcedeleon/Dat3M/blob/d8e6066d65d3ddbb3299b933b6caa6ac5a8cbc39/dartagnan/src/main/java/com/dat3m/dartagnan/program/event/Tag.java#L207) comment when `relaxed` is removed.

Reply:
I removed it.

Reply:
I will make a run with the benchmarks of the cna-verification repo and if there are no surprises we can merge

Reply:
Oh damn. We need to recompile all existing LKMM tests with the updated header. Annoying...